docs/index.md

```diff
@@ -73,9 +73,14 @@ To use the Red Hat Cloud Services provider inside your Terraform configuration y
 
   You must complete some AWS account and local configurations to create and managed ROSA clusters.
 
-* An offline [OCM token](https://console.redhat.com/openshift/token/rosa)
+* One of the following for authenticating against the OCM service:
+  * An offline [OCM token](https://console.redhat.com/openshift/token/rosa)
```

Comment:
Is this still the case? When I go to that URL, I now see the new SSO login method. I see an option to still use API keys via a link:

![image](https://github.com/user-attachments/assets/27986eac-c6ed-4c03-8605-b9f7fa757d5a)

But going there notes that the keys are deprecated:

![image](https://github.com/user-attachments/assets/5a456f55-ef9d-4782-87de-4e3f2f3bfc06)

Reply:
@fjcloud please ping me on here or preferably slack for review on that new MR :)